stable_baselines3/common/base_class.py

```diff
@@ -170,8 +167,22 @@ def __init__(
             raise ValueError("generalized State-Dependent Exploration (gSDE) can only be used with continuous actions.")
 
     @staticmethod
-    def _wrap_env(env: GymEnv, verbose: int = 0) -> VecEnv:
+    def _wrap_env(env: GymEnv, verbose: int = 0, monitor_wrapper: bool = True) -> VecEnv:
+        """ "
+        Wrap environment with the appropriate wrappers if needed.
+        For instance, to have a vectorized environment
+        or to re-order the image channels.
+
+        :param env:
+        :param verbose:
+        :param monitor_wrapper: Whether to wrap the env in a ``Monitor`` when possible.
+        :return: The wrapped environment.
+        """
         if not isinstance(env, VecEnv):
```

Comment:
What if we have a vecenv but those do not have monitor wrappers? Hard to check for this case, but could also be hard to miss.

Reply:
we would need to add a method to `VecEnv` to wrap it in all cases.

But that's why we rather provide `make_vec_env`.
This PR is mainly meant when people are using single Gym env (as in #232 )